deploy/catalog_resources/kiecloud-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,201 @@
+      mediatype: image/png
+  labels:
+    alm-owner-kiecloud: kiecloud-operator
+    operated-by: kiecloud-operator.v0.0.1
+  selector:
+    matchLabels:
+      alm-owner-kiecloud: kiecloud-operator
+      operated-by: kiecloud-operator.v0.0.1
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: true
+  customresourcedefinitions:
+    owned:
+      - version: v1
+        kind: KieApp
+        description: An instance of KieApp
+        displayName: KieApp
+        name: kieapps.app.kiegroup.org
+        metadata:
+          annotations:
+            alm-examples: >-
+              [{"apiVersion":"app.kiegroup.org/v1","kind":"KieApp","metadata":{"name":"trial"},"spec":{"environment":"trial"}}]
+        resources:
+          - kind: DeploymentConfig
+            version: apps.openshift.io/v1
+          - kind: Secret
+            version: v1
+          - kind: PersistentVolumeClaim
+            version: v1
+          - kind: ServiceAccount
+            version: v1
+          - kind: Role
+            version: authorization.openshift.io/v1
+          - kind: RoleBinding
+            version: authorization.openshift.io/v1
+          - kind: Service
+            version: v1
+          - kind: Route
+            version: route.openshift.io/v1
+        specDescriptors:
+          - description: Environment of PAM to deploy.
+            displayName: Environment
+            path: environment
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:label"
+          - description: The desired number of Kie Server Deployments for the KieApp environment
+            displayName: Kie Deployments
+            path: kieDeployments
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:podCount"
+        statusDescriptors:
+          - description: The status of each of the Deployments for the KieApp environment.
+            displayName: Deployment Status
+            path: deployments
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
+  install:
+    strategy: deployment
+    spec:
+      permissions:
```

Comment:
Wouldn't it be better if we had a `csv-gen.sh` script that would copy the contents of the permissions and deployments from the standalone yaml files? We would avoid duplicity

Reply:
i think so, all but the roles/rules as there are differences currently

Reply:
@bmozaffa @ruromero is this something we want with this PR or a future one?

Reply:
I'm eager to get this merged, so I vote for later.